### DISCUSSION TITLE
Milvus server doesn't start (timeout)

### DISCUSSION THREAD

**Discussion by asutic** [milvus-io/milvus#26894] [Q&A and General discussion]
I've installed Milvus Lite in my environment in order to understand better its capabilities. However, the service doesn't start and after a while, I see a timeout message:

`Wait for milvus server started timeout.`

Is there anything special to set up on machines that run on networks behind a proxy?

**Comment by asutic** [comment]
On the same machine, I'm able to start milvus-standalone. So, the issue might be specific to Milvus Lite.

**Reply by yhmo** [reply]
A known issue is: the previous milvus server is still running in background, the new server will get error since the address/port is occupied by the previous server, and failed to start. 

You can force close the milvus server by running a shell script with the following content:
```
echo "Stopping milvus..."
PROCESS=$(ps -e | grep milvus | grep -v grep | awk '{print $1}')
if [ -z "$PROCESS" ]; then
  echo "No milvus process"
  exit 0
fi
kill -9 $PROCESS
echo "Milvus stopped"
```

Create a script named "stop_milvus.sh", paste these lines into the script, then run this script in your terminal. It will force stop the previous Milvus server then you can start a new milvus-lite server.

**Reply by asutic** [reply]
Hi @yhmo. Thanks for you response, but the proposed solution doesn't resolve the problem. The script reports that no other milvus process exist. And my minimal example with Milvus Lite still hangs:

```
from milvus import default_server

default_server.start()
```

With debug server, the following message is printed: `error="err: failed to connect xxx.xxx.xxx.xxx:53100, reason: context deadline exceeded`.

**Reply by yhmo** [reply]
53100 is the port of the root coordinator, it indicates the rootcoord failed to start. Possible reason is the embedded etcd server is unhealthy.

There should be a directory named "test_milvus" is you set the base dir for the milvus server:
`default_server.set_base_dir('test_milvus')`

<img width="260" alt="截屏2023-09-30 23 33 06" src="https://github.com/milvus-io/milvus/assets/2282099/121b2d5b-2217-4bd7-8ba4-b95741dd22f7">

It contains a log folder which stores milvus runtime logs. There is a log file "standalone-1.log" under the "log" folder, could you show me the file?

**Reply by asutic** [reply]
I already discarded Milvus in favor of ChromaDB. If I decide to go back to Milvus in the future, I'll post my findings here.
